workers/pyproject.toml

```diff
@@ -15,7 +15,7 @@ keywords = [
   ]
 
 [tool.poetry.dependencies]
-python = "^3.8"
+python = "~3.8"
```

Comment:
Why this change? All the other dependencies are defined with the caret and not tilde. Is there an issue with Python 3.9?

Reply:
With `~3.8` I got following error when running `poetry`.
```
  The current project's Python requirement (>=3.8,<4.0) is not compatible with some of the required packages Python requirement:
    - scipy requires Python >=3.7,<3.11, so it will not be satisfied for Python >=3.11,<4.0

  Because scipy (1.7.3) requires Python >=3.7,<3.11
   and no versions of scipy match >1.7.3,<2.0.0, scipy is forbidden.
  So, because ohsome-quality-analyst depends on scipy (^1.7.3), version solving failed.
  
  [ ... ]
  
  
  ```
  So I though it would be a good idea to fix the version to a specific minor version. But now I chose to use following version requirement:

Reply:
With `~3.8` I got following error when running `poetry`.
```
  The current project's Python requirement (>=3.8,<4.0) is not compatible with some of the required packages Python requirement:
    - scipy requires Python >=3.7,<3.11, so it will not be satisfied for Python >=3.11,<4.0

  Because scipy (1.7.3) requires Python >=3.7,<3.11
   and no versions of scipy match >1.7.3,<2.0.0, scipy is forbidden.
  So, because ohsome-quality-analyst depends on scipy (^1.7.3), version solving failed.
  
  [ ... ]
  
  ```
  So I though it would be a good idea to fix the version to a specific minor version. But now I chose to use following version requirement:

Reply:
maybe add a comment to it, then :white_check_mark:

Reply:
Added a comment to pyproject.toml.